rspace/src/main/scala/coop/rchain/rspace/history/HistoryAction.scala

```diff
@@ -1,11 +1,10 @@
 package coop.rchain.rspace.history
 
 import coop.rchain.rspace.hashing.Blake2b256Hash
-import coop.rchain.rspace.history.History.KeyPath
 
 sealed trait HistoryAction {
-  def key: KeyPath
+  def key: KeySegment
 }
 
-final case class InsertAction(key: KeyPath, hash: Blake2b256Hash) extends HistoryAction
-final case class DeleteAction(key: KeyPath)                       extends HistoryAction
+final case class InsertAction(key: KeySegment, hash: Blake2b256Hash) extends HistoryAction
```

Comment:
It looks like base trait is not used anywhere except when defining `KeySegment` implementation. Why to define trait then at all?